src/modules/esl-carousel/core/esl-carousel.ts

```diff
@@ -306,6 +306,15 @@ export class ESLCarousel extends ESLBaseElement {
     return el && el.hasAttribute('prev');
   }
 
+  // Navigates to requested slide
+  @listen('esl:show:request')
+  protected onHandleFootnotes(e: CustomEvent): void {
+    const target = (e.target as HTMLElement).closest('[esl-carousel-slide]') as HTMLElement;
+    const index = this.$slides.indexOf(target);
+    if (index === -1) return;
+    this.goTo(`slide:${index}`);
```

Comment:
Should be fine unless we have a bug
```suggestion
    if (index !== -1) this.goTo(index);
```